plugins/main/bridge/bridge_test.go

```diff
@@ -1672,6 +1764,25 @@ var _ = Describe("bridge Operations", func() {
 		Expect(testutils.UnmountNS(targetNS)).To(Succeed())
 	})
 
+	var (
+		correctMinID int = 100
+		correctMaxID int = 105
+	)
+
+	DescribeTable(
+		"collectVlanTrunk succeeds",
+		func(vlanTrunks []*VlanTrunk, expectedVIDs []int) {
+			Expect(collectVlanTrunk(vlanTrunks)).To(Equal(expectedVIDs))
```

Comment:
```suggestion
			Expect(collectVlanTrunk(vlanTrunks)).To(ConsistOf(expectedVIDs))
```

otherwise you get the following error:
```
  [FAILED] Expected
      <[]int | len:0, cap:0>: nil
  to equal
      <[]int | len:0, cap:0>: []
  In [It] at: /home/mduarted/github/cni-plugins/plugins/main/bridge/bridge_test.go:1775
```

Reply:
Thanks for advice
this one is better.

Reply:
what you had (ensure the return val is `nil` is also OK). You can use it if you prefer.